src/components/site/forms/table/index.jsx

```diff
@@ -56,7 +57,7 @@ class Table extends React.PureComponent {
                           'row-title': includeRowTitles && j === 0,
                         })}
                       >
-                        {col}
+                        <Trans>{col}</Trans>
```

Comment:
this can result in an attempted double translation, some child content in `col` is just plaintext that has been unprocessed but in other instances it is wrapped in `<span>` tags at the `ObservationTable` level. There's no real risk to trying to parse it twice but if debugging is turned on it will throw a key-not-found warning.